lib/ansible/modules/git.py

```diff
@@ -438,6 +447,21 @@ def get_version(module, git_path, dest, ref="HEAD"):
     return sha
 
 
+def ssh_supports_acceptnewhostkey(module):
+    try:
+        ssh_path = module.get_bin_path('ssh', True)
```

Comment:
`module.get_bin_path()` does not raise an exception but handles it internally.

Use `ansible.module_utils.common.process.get_bin_path` instead.

```suggestion
        ssh_path = get_bin_path('ssh')
```

Reply:
Fixed in https://github.com/ansible/ansible/pull/70499/commits/9f0c86fa715cd78b5c3fed6fc62b8290b267e541.

Reply:
Also imported the function in https://github.com/ansible/ansible/pull/70499/commits/1ac129d37ddaae1c2dc350457e825c861f2b31fe.